app/src/main/java/com/hover/stax/contacts/PhoneHelper.java

```diff
@@ -12,19 +12,6 @@
 public class PhoneHelper {
     final static private String TAG = "PhoneHelper";
 
-    public static String getNumberFormatForInput(String accountNumber, HoverAction a, Channel c) {
```

Comment:
Don't pass complex objects to a helper function. Now it has to know about the internals of 2 objects